zerver/openapi/zulip.yaml

```diff
@@ -1869,6 +1869,49 @@ paths:
                                     [["Denmark", "topic", 1594825442]],
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event sent to a user's clients when that user's set of
+                                configured user-topic relationships have changed.
```

Comment:
Unlike the `muted_topics` array, this only returns the changed / updated user-topic pair, correct? So maybe ...

> ... when one of that user's configured user-topic relationships has changed.

Reply:
Plus, you'll want to add a **Changes** note here with the feature level. The `muted_users` event is nearby in the documentation and can be used as an example.